app/src/renderer/components/monitor/PageBlock.vue

```diff
@@ -38,29 +38,41 @@ page(:title="`Block ${block.header.height}`")
     :dd="p.signature.data")
 
   part(title='Transactions')
-    list-item(v-for="tx in block.data.txs" :key="tx.id"
-      dt="Transaction" :dd="TODO")
+    list-item(v-if="block.header.num_txs > 0" v-for="tx in block.data.txs" :key="tx.id" dt="Transaction" :dd="TODO")
```

Comment:
use `li-transactions`?

Reply:
It should probably show more information at some point, but so should the information in the transactions list.